.vscode/settings.json

```diff
@@ -1,9 +1,10 @@
-{
-  "editor.formatOnPaste": true,
-  "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.fixAll.format": true
-  }
-}
+{
+  "editor.formatOnPaste": true,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true,
+    "source.fixAll.format": true
+  },
+  "typescript.tsdk": "node_modules\\typescript\\lib"

```

Comment:
Why was this line of code added to specify the development tool package path for TypeScript? Is there any specific content-related requirement for it?